MaxText/configs/base.yml

```diff
@@ -533,6 +533,7 @@ inference_metadata_file: "" # path to a json file
 inference_server: "MaxtextInterleavedServer"  # inference server to start
 inference_benchmark_test: False
 enable_model_warmup: False
+multi_sampling: False
```

Comment:
Should we start breaking apart inference vs. training vs. generic config @A9isha @vipannalla (as a follow up if yes)